src/Mod/Start/Gui/StartView.cpp

```diff
@@ -392,6 +392,11 @@ void StartView::newArchFile() const
     catch (...) {
         Gui::Application::Instance->activateWorkbench("ArchWorkbench");
     }
+
+    // Set the camera zoom level to 20 m, which is more appropriate for architectural projects
+    Gui::Command::doCommand(
+        Gui::Command::Gui,
+        "Gui.activeDocument().activeView().viewDefaultOrientation(\"Isometric\", 20000.0)");

```

Comment:
```suggestion
        "Gui.activeDocument().activeView().viewDefaultOrientation(\"Trimetric\", 10000.0)");

```
Change defaults  as per discussion on PR.